src/Servers/Kestrel/Core/src/Internal/Infrastructure/HttpCharacters.cs

```diff
@@ -182,6 +182,7 @@ public static int IndexOfInvalidTokenChar(ReadOnlySpan<byte> span)
             return -1;
         }
 
+        // Disallows control characters and anything more than 0x7E
```

Comment:
```suggestion
        // Disallows control characters and anything more than 0x7F
```
?

Reply:
😁 7E is also a prohibited control character (Delete), but I will change the comment to be consistent.